### PR TITLE
Update CI links

### PR DESCRIPTION
**Pull request by cofyc**
fixes https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/issues/28

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -277,22 +277,8 @@ to "Block".
...
-
-[GCE Serial](https://k8s-testgrid.appspot.com/sig-storage#gce-serial&include-filter-by-regex=PersistentVolumes-local)
-
-[GKE Serial](https://k8s-testgrid.appspot.com/sig-storage#gke-serial&include-filter-by-regex=PersistentVolumes-local)
-
-[GCE Alpha](https://k8s-testgrid.appspot.com/sig-storage#gce-alpha&include-filter-by-regex=PersistentVolumes-local)
-
-[GKE Alpha](https://k8s-testgrid.appspot.com/sig-storage#gke-alpha&include-filter-by-regex=PersistentVolumes-local)
 
+Check testgrid [sig-storage-local-static-provisioner](https://testgrid.k8s.io/sig-storage-local-static-provisioner) dashboard.
```
Comment (msau42) [inline README.md]: I noticed testgrid results are not listing the test cases.  I think you need to pass through the ARTIFACTS env variable to get it to work:
https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/184

Comment (msau42) [reply README.md]: Also is this path going to be correct after https://github.com/kubernetes/test-infra/pull/10696?

Comment (cofyc) [reply README.md]: Path will be correct.

I'll take a look at ARTIFACTS.

**Pull request merged**